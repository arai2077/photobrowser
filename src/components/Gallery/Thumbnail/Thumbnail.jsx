import React from 'react';

const Thumbnail = ({ photo }) => (
  <img src={photo.thumbnailUrl} />
)

export default Thumbnail;