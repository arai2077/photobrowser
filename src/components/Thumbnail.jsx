import React from "react";

const Thumbnail = ({ id, thumbnailUrl, title }) => (
	<img key={id} src={thumbnailUrl} alt={title} />
);

export default Thumbnail;
