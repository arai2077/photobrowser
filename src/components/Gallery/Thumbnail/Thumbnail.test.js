import React from "react";
import { shallow } from "enzyme";
import Thumbnail from "./Thumbnail";

describe("Thumbnail", () => {
  it("renders", () => {
    const component = shallow(<Thumbnail />);

    expect(component).toMatchSnapshot();
  });
});
