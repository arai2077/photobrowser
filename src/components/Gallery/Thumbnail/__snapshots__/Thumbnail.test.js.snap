// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Thumbnail renders 1`] = `
<Fragment>
  Some thumbnail
</Fragment>
`;
